assets/dev/js/editor/elements/views/container.js

```diff
@@ -600,6 +604,10 @@ const ContainerView = BaseElementView.extend( {
 			this.handleGridEmptyView();
 		}
 	},
+
+	isDistortsPreview() {
+		return this.container.children.length === this.container.settings.get( 'grid_columns_grid' ).size * this.container.settings.get( 'grid_rows_grid' ).size;
```

Comment:
@mykytamurzin Need to handle current device mode etc `grid_rows_grid_tablet` etc. Check how it's done in `getDeviceGridDimensions()`.